.github/workflows/e2e-arm64-template.yaml

```diff
@@ -12,10 +12,6 @@ jobs:
     # this is to prevent the job to run at forked projects
     if: github.repository == 'etcd-io/etcd'
     runs-on: actuated-arm64-8cpu-32gb
-    container: golang:1.21-bookworm
```

Comment:
👍 

Hopefully this will also yield an improvement

Reply:
Certainly do, see line 26 :)